spec/services/accept_unconditional_offer_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+RSpec.describe AcceptUnconditionalOffer do
+  include CourseOptionHelpers
+
+  it 'sets the accepted_at date for the application_choice' do
```

Comment:
can we also add a test for `recruited_at` timestamp?

Reply:
:+1: https://github.com/DFE-Digital/apply-for-teacher-training/pull/4154/commits/076ccbcee103bd0dd06383c5f04761f62314d8f6